README.md

```diff
@@ -0,0 +1 @@
+Use `composer test` for running tests
```

Comment:
Let's write readme with basic description, installation, example usage in a few scenarios, and similar stuff. Compare with 5 other utopia libraries of your choice, to make sure we are not missing anything in main readne